CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # App Center SDK for React Native Change Log
 
+## Version 3.1.2
+
+### App Center Crashes
+
+#### Android
+
+* **[Fix]** Fix sending attachments with a `null` text value.
```

Comment:
Does it really affects React-Native wrapper?

Reply:
I've checked it and yes, it affects our RN SDK.
It can be checked from the Crashes Listener code if you change the first argument to null:
```
if (textAttachment !== null) {
attachments.push(ErrorAttachmentLog.attachmentWithText(null, 'hello.txt'));
}
```
After the Android fix, it is handled correctly on the RN side.